src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,91 @@
 package core.basesyntax;
 
+import java.util.ArrayList;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    static final int DEFAULT_INITIAL_CAPACITY = 16;
+    static final int MAXIMAL_CAPACITY = 1 << 30;
+    static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private int size = 0;
+
+    private ArrayList<Entry<K,V>>[]table = new ArrayList[DEFAULT_INITIAL_CAPACITY];
 
     @Override
     public void put(K key, V value) {
+        int index = hash(key);
+
+        if (index >= 0) {
+            if (size >= DEFAULT_LOAD_FACTOR * table.length) {
```

Comment:
The condition to resize the table should check against the threshold, which is `table.length * DEFAULT_LOAD_FACTOR`. Consider storing this value in a separate field and updating it in the `resize` method.

Reply:
ok, done